bot/code_review_bot/__init__.py

```diff
@@ -135,21 +139,31 @@ def is_publishable(self):
         if self.level == Level.Error:
             return True
 
+        # Issues that are new are reported
+        if self.new_issue:
+            return True
+
         # Then check if the backend marks this issue as publishable
         if self.on_backend is not None:
             return self.on_backend["publishable"]
 
         # Fallback to in_patch detection
+        return self.is_in_patch()
+
+    def is_in_patch(self):
         return self.revision.contains(self)
 
-    def build_hash(self, local_repository=None):
+    def get_hash(self, local_repository=None):
```

Comment:
It would be nicer to use [cached_property](https://docs.python.org/3/library/functools.html#functools.cached_property) and moving `local_repository` to a setting